frontend/smart-kickers-game/src/apis/getStatistics.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
+import config from '../config';
+
+export async function getStatistics() {
+  try {
+    const result = await axios.get(`${config.apiBaseUrl}/stats`);
+
+    return {
+      data: result.data,
+    };
```

Comment:
```suggestion
    return result.data;
```

As the response is a JSON object, this could work better

Reply:
Corrected